packages/eslint-plugin-template/src/configs/accessibility.json

```diff
@@ -10,6 +10,7 @@
     "@angular-eslint/template/mouse-events-have-key-events": "error",
     "@angular-eslint/template/no-autofocus": "error",
     "@angular-eslint/template/no-distracting-elements": "error",
+    "@angular-eslint/template/prefer-ngsrc": "error",
```

Comment:
Please remove this again, it is what is causing the integration tests to fail

A) you should not manually patch this file, it is generated based on using a prefix in the rule description
B) we will not be modifying the accessibility config in minor versions. We will only do that in major versions
C) I don't believe this even is an accessibility rule? It's a web performance rule?

Reply:
Fixed.